go-controller/pkg/ovn/master.go

```diff
@@ -483,6 +483,11 @@ func (oc *Controller) syncNodeManagementPort(node *kapi.Node, hostSubnets []*net
 			return err
 		}
 
+		// Non existing Services and Services without endpoints are rejected by default
```

Comment:
Is this commit required for endpoint slices? If not let's remove it and do another PR. This one is already huge.

Reply:
this is not required, we talk about reusing current way of handling reject ACLs